oktapam/constants/descriptions/attributes.go

```diff
@@ -89,6 +90,8 @@ var (
 	LocalAccount                              = "The username of a local account on a server."
 	LocalAccounts                             = "The usernames of a local accounts on a server."
 	ManagedPrivilegedAccounts                 = "An array of managed accounts for password rotation."
+	MFAACRValues                              = "The authentication context class reference value used to defines a specific set of assurance level requirements that the protected resource requires."
```

Comment:
@isaacdurham-okta - could you take a look at these descriptions?

Reply:
```suggestion
	MFAACRValues                              = "The authentication context class reference (ACR) for this policy. This defines a specific set of assurance level requirements required by a protected resource."
```